lib/game.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class Game
```

Comment:
I have no issues with this, but I might approach this class differently, what do you think if I were to make some changes like this?

```ruby
class Game
  class << self
    def from_beckon(beckon)
     players = beckon #somehow get from beckon
      new(
        # i feel like the `Team` class should be the one used to draft teams
        Team.from_player_list(players) # this could generate two teams [Team, Team],
        ["Dust II"] # add generated list here
      )
    end
  end

  def initialize(teams, maps)
    @teams = teams
    @maps = maps
  end
end
```